database/config.json

```diff
@@ -1,14 +1,14 @@
 {
     "development": {
-        "username": "student",
-        "password": null,
+        "username": "root",
```

Comment:
Group note: update development object to have "student" as username and password as "student" acc: to original master repo.